inf-clojure.el

```diff
@@ -725,19 +731,24 @@ The value is nil if it can't find one."
 (defun inf-clojure-show-var-documentation (var)
   "Send a form to the inferior Clojure to give documentation for VAR.
 See function `inf-clojure-var-doc-form'."
-  (interactive (inf-clojure-symprompt "Var doc" (inf-clojure-var-at-pt)))
-  (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-doc-form) var)))
+  (interactive "P")
+  (let ((var (if (inf-clojure--prompt-on-commands-p var)
+                 (car (inf-clojure-symprompt "Var doc" (inf-clojure-var-at-pt)))
+               (inf-clojure-var-at-pt))))
+    (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-doc-form) var))))
 
 (defun inf-clojure-show-var-source (var)
-  "Send a form to the inferior Clojure to give source for VAR.
-See variable `inf-clojure-var-source-form'."
-  (interactive (inf-clojure-symprompt "Var source" (inf-clojure-var-at-pt)))
-  (comint-proc-query (inf-clojure-proc) (format inf-clojure-var-source-form var)))
+  "Send a command to the inferior Clojure to give source for VAR.
+See variable `inf-clojure-var-source-command'."
+  (interactive "P")
+  (let ((var (if (inf-clojure--prompt-on-commands-p var)
```

Comment:
Sorry to bother on this again, I am reading [this](https://www.gnu.org/software/emacs/manual/html_node/elisp/Local-Variables.html) and I really don't see any difference with Clojure and no reason why the above is wrong:

> A variable can have more than one local binding at a time (e.g., if there are nested let forms that bind the variable). The current binding is the local binding that is actually in effect. It determines the value returned by evaluating the variable symbol, and it is the binding acted on by setq.

Again, you ae the boss so I will rename it 😀